### ISSUE TITLE
Wait for alert keyword fails at wait for promise 

### ISSUE THREAD

**Issue by IDvoe** [MarketSquare/robotframework-browser#1980]
With Browser 12.3.0, when verifying the step wait for ${promsie} fails with this error 'str' object has no attribute 'name' 

**To Reproduce**
```
Test to check alert
    [Arguments]    ${locator}    ${action}    ${text}
    ${promise}    promise to    wait for alert    action=${action}    text=${text}
    click    ${locator}
    wait for    ${promise}    # this is where it fails with  'str' object has no attribute 'name' 
```

**Expected behavior**
wait for alert should not fail 

**Screenshots**
Stack trace screenshot 
![image](https://user-images.githubusercontent.com/104168783/164546486-51c768ba-1588-48d5-a048-133978bb0247.png)

**Desktop (please complete the following information):**
 - OS:  iOS (BigSur, intel core)
 - Browser: chrome

**Comment by aaltat** [comment]
@allcontributors please add @IDvoe for bugs.

**Comment by aaltat** [comment]
This looks pretty bad. As a workaround, could you try: 
`${promise}    promise to    wait for alert    ${action}    ${text}`

**Comment by IDvoe** [comment]
The workaround doesn't work. I'm using handle future dialogs for time being. Hope you can fix the issue soon.

**Comment by luchoagomezt** [comment]
> The workaround doesn't work. I'm using handle future dialogs for time being. Hope you can fix the issue soon.

Hello IDvoe,
Could you please paste the Robot logs, one similar to the following example:

**Comment by IDvoe** [comment]
Here are the logs
<img width="1151" alt="image" src="https://user-images.githubusercontent.com/104168783/166409898-25fcd68f-1e2d-4ed7-bafa-b716880ea49b.png">

**Comment by luchoagomezt** [comment]
@IDvoe, @aaltat 
I was able to reproduce the issue,  as the value for the name argument **action** is passed as '**ACCEPT**' instead of '**accept**', the code does not assign the object DialogAction.accept but instead the string 'ACCEPT'.

**Comment by IDvoe** [comment]
Thank you, with the browser library 9 the action=ACCEPT worked fine only after updating to 12.3.0, the test started failing.  can I  use both lower case and upper case for accept when the code fix is released? Thanks once again.

**Comment by luchoagomezt** [comment]
> Thank you, with the browser library 9 the action=ACCEPT worked fine only after updating to 12.3.0, the test started failing. can I use both lower case and upper case for accept when the code fix is released? Thanks once again.

It is a simple fix, as of now only action=accept works, but I will fix the line of code.

**Comment by luchoagomezt** [comment]
PR  at https://github.com/MarketSquare/robotframework-browser/pull/2009

**Comment by aaltat** [comment]
Fixed at #2009
